sklearn_xarray/preprocessing.py

```diff
@@ -1593,3 +1613,134 @@ def reduce(X, return_estimator=False, **fit_params):
         return Xt, estimator
     else:
         return Xt
+
+    
+ 
+class Stackerizer(BaseTransformer):
```

Comment:
I think `Stacker` would be a more fitting name
```suggestion
class Stacker(BaseTransformer):
```